dev/com.ibm.ws.install/src/com/ibm/ws/install/internal/ArtifactDownloader.java

```diff
@@ -185,9 +184,6 @@ public void synthesizeAndDownload(String mavenCoords, String filetype, String dL
         checksumFormats[1] = "SHA1";
         checksumFormats[2] = "SHA256";
 
-        checkValidProxy();
```

Comment:
Authentications will be configured in line 198 (when we call `testConnection(repository)`).